src/Aspire.Hosting.Azure/Aspire.Hosting.Azure.csproj

```diff
@@ -40,6 +41,8 @@
 
   <ItemGroup>
     <InternalsVisibleTo Include="Aspire.Hosting.Azure.Tests" />
+    <InternalsVisibleTo Include="Aspire.Hosting.Azure.AppContainers" />
+    <InternalsVisibleTo Include="Aspire.Hosting.Azure.Storage" />
```

Comment:
We can't use IVT because people can use different versions, which they can then get into a torn state.